docs/en/setup/plugins/forwarder_nativemeter-grpc-forwarder.md

```diff
@@ -2,8 +2,15 @@
 ## Description
 This is a synchronization meter grpc forwarder with the SkyWalking meter protocol.
 ## DefaultConfig
-```yaml```
+```yaml
+# The upstream LRU Cache max size
+upstream_lru_cache_size: 5000
+# The upstream LRU Cache time(second) on each service instance
+upstream_lru_cache_ttl: 180
+```
 ## Configuration
 |Name|Type|Description|
 |----|----|-----------|
+| upstream_lru_cache_size | int | The upstream LRU Cache max size |
+| upstream_lru_cache_ttl | int | The upstream LRU Cache time(second) on each service instance |
```

Comment:
I am reading again, feel like these two names are misleading. I think we are not just caching `upstream`, the real data in the cache is the `routine_rule` of meter data.

```suggestion
| routine_rule_lru_cache_size | int | The LRU policy cache size for hosting routine rules of service instance. |
| routine_rule_lru_cache_ttl | int | The TTL of the LRU cache size for hosting routine rules of service instance. |
```